features/insured/individual_signup.feature

```diff
@@ -3,52 +3,21 @@ Feature: Insured Plan Shopping on Individual market
   Background:
     Given Individual has not signed up as an HBX user
     Given the FAA feature configuration is enabled
-    When Individual visits the Insured portal during open enrollment
+    And Individual market is under open_enrollment period
     Then Individual creates a new HBX account
     Then Individual should see a successful sign up message
```

Comment:
Why are we still calling these steps? It seems like we're correctly creating the `Patrick Doe` user below, on line 11.

Reply:
Yeah, we don't need that, removed in this commit- https://github.com/ideacrew/enroll/pull/545/commits/23af69a48ba11668c8aca22f8ee9e4abf9a92e0d